utils/gen_onnx_mlir.py

```diff
@@ -282,6 +282,7 @@
     'GlobalAveragePool',
     'GlobalMaxPool',
     'Identity',
+    'PRelu',
```

Comment:
This should be added to `OpsWithVerifier` list instead of `OpsWithCanonicalizer` list.